index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+ <div id="main"> 
+  <nav>
+    <div>
+      <h1 id="title">Bjarne Stroustrup</h1>
+      <h3>He is the creator of C++ programming language</h3>
+    </div>
+  </nav>
+  <section class="images-section">
+    <div class="images" id="img-div">
+      <img id="image" src="images/Bjarne_Stroustrup_UC3M_2-1.jpg" alt="">
+      <p id="img-caption">Stroustrup was born in Aarhus, Denmark. His family was working class, and he went to the local schools.[7] He attended Aarhus University 1969–1975 and graduated with a master's degree in mathematics and computer science. His interests focused on microprogramming and machine architecture. He learned the fundamentals of object-oriented programming from its inventor, Kristen Nygaard, who frequently visited Aarhus.</p>
+    </div>
+  </section>
+  <section id="tribute-info">
+    <div>
+      <h1>Here's a time line of Bjarne Stroustrup:</h1>
+      <div class="list-item">
+        <ul>
+          <li><span class="year">In 1979 -</span> Stroustrup began his career as a member of technical staff in the Computer Science Research Center of Bell Labs in Murray Hill, New Jersey, USA.</li>
+          <li><span class="year">1993 -</span>He was made a Bell Labs fellow and in 1996, an AT&T Fellow.</li>
+          <li><span class="year">2002 to 2014 -</span>Stroustrup was the College of Engineering Chair Professor in Computer Science at Texas A&M UniversityFrom 2011</li>
+          <li><span class="year">2011 -</span>he was made a University Distinguished Professor.</li>
+          <li><span class="year">2014 - </span>Stroustrup is a Technical Fellow and Managing Director in the technology division of Morgan Stanley in New York City and a Visiting Professor in Computer Science at Columbia University</li>
+        </ul>
+        
+      </div>
+    </div>
+  </section>
+  <section id="footer">
```

Comment:
Although It works, semantically it's better to use a ```footer``` tag instead of a ```div```.
by using semantic tags, we make better and more comprehensible HTML by defining different sections of our webpage, this helps not only other developers to read your code in a more manageable way it makes it easier for web browsers to interpret your content.